ops/envs/geth.env

```diff
@@ -29,5 +29,5 @@ IPC_DISABLE=true
 NETWORK_ID=420
 NO_USB=true
 NO_DISCOVER=true
-TARGET_GAS_LIMIT=9000000
+TARGET_GAS_LIMIT=10000000
```

Comment:
Just a question on how the update to `ExecutionManager.maxTransactionGasLimit` was done [here](https://github.com/ethereum-optimism/optimism/pull/840/files#diff-17b617e193a2e9627cf0c5459c05a94f7de9ce2ebde734ede16251f1a7a16118R329) since I am reading `maxTransactionGasLimit` being stored in slot `2` not `4`. What am I missing?

Reply:
We spoke out of band and have agreed that the slot is `4`, see https://github.com/ethereum-optimism/optimism/pull/840